packages/react/src/components/Video/VideoBase.tsx

```diff
@@ -0,0 +1,102 @@
+import { MutableRefObject, useEffect, useState } from "react";
+import { Video } from "@signalwire/js";
+
+interface IVideoRequiredProps {
+  token: string;
+}
+
+interface IVideoOptionalProps {
+  logLevel?: "trace" | "debug" | "info" | "warn" | "error" | "silent";
+  audio?: boolean;
+  video?: boolean;
+  children?: JSX.Element;
+  rootElement?: MutableRefObject<HTMLElement | null>;
```

Comment:
could this type be just `rootElement?: HTMLElement;`?

Reply:
I don't think so. 

Changed to `RefObject<HTMLDivElement>`. We'll want to pass a reference rather than `ref.current` here as `ref.current` is null on start, and since it's a ref, it's change doesn't trigger a rerender.